docs/sources/installation/configuration.md

```diff
@@ -34,6 +34,8 @@ A common problem is forgetting to uncomment a line in the `custom.ini` (or `graf
 > `/etc/grafana/grafana.ini`. This path is specified in the Grafana
 > init.d script using `--config` file parameter.
 
+**Mac:** By default, the Configuration file is located at `/usr/local/etc/grafana/grafana.ini`.
```

Comment:
```suggestion
**macOS:** By default, the configuration file is located at `/usr/local/etc/grafana/grafana.ini`.
```

Maybe we should restructure this section a bit. The default locations vary depending on how they were installed. 

If you downloaded the binaries yourself, the default is `$WORKING_DIR/conf/defaults.ini`.
If you installed using `deb` or `rpm` packages on Linux, the config file is in `/etc/grafana/grafana.ini`.
If you installed using `brew` on macOS, then `/usr/local/etc/grafana/grafana.ini`
If you installed on Windows, then it seems to be same as the binaries.

Reply:
Sounds good. If we don't want to have this on multiple places, maybe we can update each installation page and link to them? For example linking to https://grafana.com/docs/installation/rpm/#package-details for rpm.

Reply:
In the longer term, I am hoping to consolidate and simplify the Installation and Getting Started documentation, probably by having merging the sections and organizing them by system:
* Getting started with Windows
* Getting Started with MacOS
* And so on

ALL of this needs to be rewritten. This first PR is just trying to begin sorting content into the right places, and I am starting with the obvious, easy stuff. For example, configuration information should not be in the installation docs. Once everything is sorted into its proper "bucket," then we can start making decisions about how else to organize it.